src/views/ResourceManagement/FieldCoreOverride/FieldCoreOverride.vue

```diff
@@ -1,60 +1,53 @@
 <template>
-  <b-container fluid="xl">
+  <BContainer fluid="xl">
     <page-title
       :title="$t('appPageTitle.fieldCoreOverride')"
       :description="$t('pageFieldCoreOverride.pageDescription')"
     />
-    <b-row>
-      <b-col md="8" xl="6">
+    <BRow>
+      <BCol md="8" xl="6">
         <alert variant="info" class="mb-5">
-          <p class="mb-0 font-weight-bold">
+          <p class="mb-0 fw-bold">
             {{ $t('pageFieldCoreOverride.alert.title') }}
           </p>
           <p>
             {{ $t('pageFieldCoreOverride.alert.description') }}
           </p>
         </alert>
-      </b-col>
-    </b-row>
+      </BCol>
+    </BRow>
 
     <current-configuration />
     <change-configuration />
-  </b-container>
+  </BContainer>
 </template>
 
-<script>
-import PageTitle from '@/components/Global/PageTitle';
-import LoadingBarMixin, { loading } from '@/components/Mixins/LoadingBarMixin';
-import Alert from '@/components/Global/Alert';
+<script setup>
+import { onBeforeMount } from 'vue';
+import { onBeforeRouteLeave } from 'vue-router';
+import useLoadingBar from '@/components/Composables/useLoadingBarComposable';
+import PageTitle from '@/components/Global/PageTitle.vue';
+import Alert from '@/components/Global/Alert.vue';
+import { SystemStore, FieldCoreOverrideStore, LicenseStore } from '@/store';
+import CurrentConfiguration from './FieldCoreOverrideInfo.vue';
+import ChangeConfiguration from './FieldCoreOverrideConfiguration.vue';
 
-import FieldCoreOverrideInfo from './FieldCoreOverrideInfo.vue';
-import FieldCoreOverrideConfiguration from './FieldCoreOverrideConfiguration.vue';
+const { startLoader, endLoader } = useLoadingBar();
 
-export default {
-  name: 'FieldCoreOverride',
-  components: {
-    PageTitle,
-    Alert,
-    CurrentConfiguration: FieldCoreOverrideInfo,
-    ChangeConfiguration: FieldCoreOverrideConfiguration,
-  },
-  mixins: [LoadingBarMixin],
-  beforeRouteLeave(to, from, next) {
-    this.hideLoader();
-    next();
-  },
-  data() {
-    return {
-      loading,
-    };
-  },
-  created() {
-    this.startLoader();
-    Promise.all([
-      this.$store.dispatch('licenses/getLicenses'),
-      this.$store.dispatch('system/getSystem'),
-      this.$store.dispatch('fieldCoreOverride/getBiosAttributes'),
-    ]).finally(() => this.endLoader());
-  },
-};
+const systemStore = SystemStore();
+const fieldCoreOverrideStore = FieldCoreOverrideStore();
+const licenseStore = LicenseStore();
+
+onBeforeMount(() => {
+  startLoader();
+  Promise.all([
+    licenseStore.getLicenses(),
+    systemStore.getSystem(),
+    fieldCoreOverrideStore.getBiosAttributes(),
+  ]).finally(() => endLoader());
+});
+
+onBeforeRouteLeave(() => {
+  hideLoader();
```

Comment:
hideLoader is not defined, After coming to FCO page, unable to go to any other page..